tutorials/iot/ubuntu-kiosk/wayland-kiosk.md

```diff
@@ -774,11 +744,11 @@ If your target device has the same CPU architecture as your PC, you can just bui
 
 
 ```bash
-snapcraft cleanbuild
+snapcraft
 ```
 
 
-However you need to build the snap for a different CPU architecture, it is not quite as simple..
+However you need to build the snap for a different CPU architecture, it is not quite as simple…
 
 
 ### Building snaps using Launchpad
```

Comment:
below, the [https://docs.snapcraft.io/build-snaps/ci-integration](https://docs.snapcraft.io/build-snaps/ci-integration) link doesn't work. This page was part of the pre-Discourse docs and there isn't a current equivalent. Maybe [https://tutorials.ubuntu.com/tutorial/continuous-snap-delivery-from-travis-ci](https://tutorials.ubuntu.com/tutorial/continuous-snap-delivery-from-travis-ci)?

The same applies to the `x11-kiosk` tutorial.

Reply:
Unfortunately that Travis guide isn't quite suitable, as it doesn't mention cross-compiling snaps at all. Closest guide I can find now is https://kyrofa.com/posts/building-your-snap-on-device-there-s-a-better-way
Will that do?

Reply:
@gerboland the problem to solve is building for a different architecture to your desktop: that only needs cross-compilation if it happens locally.

This can be done by using a build system that supports your architecture.

* For Mir we use launchpad (but the instructions have indeed disappeared)
* There's also https://snapcraft.io/build - AFAICS that doesn't require registering the snap with the store.
* I assume travis can also do this with a suitable builder, but that tutorial doesn't make it explicit.
* Ideally, we'd have a command line like: https://forum.snapcraft.io/t/preview-snapcraft-remote-build/11541 (but that isn't available yet)

I suggest checking out https://snapcraft.io/build and (if viable) using that.

Reply:
I've re-written this to refer to the snapcraft.io/build service, I'd appreciate a sanity check